README.md

```diff
@@ -30,6 +31,15 @@ pip install squidasm --extra-index-url=https://{netsquid-user-name}:{netsquid-pa
 
 Now everything should be setup and ready in order to create your own applications and experiments and run them on the simulator!
 
+### Troubleshooting
```

Comment:
Please add this paragraph to docs/introduction.rst too. The format for headings is a bit different.
(docs directory contains the input for generating the readthedocs documentation).

Reply:
Updated introduction.rst